common/utils/src/main/resources/error/error-conditions.json

```diff
@@ -4830,6 +4830,11 @@
           "<variableName> is a VARIABLE and cannot be updated using the SET statement. Use SET VARIABLE <variableName> = ... instead."
         ]
       },
+      "SQL_SCRIPTING_NOT_ENABLED" : {
```

Comment:
The words `NOT_ENABLED` are not needed, just leave `SQL_SCRIPTING`.

Reply:
Done.